src/common/types/proxy-types.ts

```diff
@@ -26,3 +26,8 @@ export async function wait(ms: number) {
     }, ms)
   })
 }
+
+export interface OverlayWindowState {
+  ignoreInput: boolean
+  state?: 'show' | 'hide' | null
```

Comment:
it needs to switch from null to hide to pass the 'hide' value from the backend to the frontend. if it's 'hide' by default and then we tell it to 'hide', it will not send to the frontend because it's the same value

Reply:
I don't think we can init to undefined because we have to set up a listener to it. I don't think that the observable will be observed if it's undefined

Reply:
Interesting, in that case maybe the state should not be optional but nullable.